src/styles/layout/desktop/index.njk

```diff
@@ -0,0 +1,17 @@
+---
+title: Two-thirds – Layout
```

Comment:
I think this title needs updating to better describe what this example is showing.